hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -106,13 +106,14 @@ public class OperationRunnerImpl extends OperationRunner implements StaticMetric
 
     static final int AD_HOC_PARTITION_ID = -2;
 
+    final OperationServiceImpl operationService;
+    final Counter executedOperationsCounter;
```

Comment:
Can you also move the `@Probe(name = OPERATION_METRIC_OPERATION_RUNNER_EXECUTED_OPERATIONS_COUNT, level = DEBUG)`  line above this field? The annotation is now on `thisAddress` field and all tests fail.